examples/large_models/trt_llm/llama/model-config.yaml

```diff
@@ -0,0 +1,12 @@
+# TorchServe frontend parameters
+minWorkers: 1
+maxWorkers: 1
+maxBatchDelay: 100
+responseTimeout: 1200
+deviceType: "gpu"
+asyncCommunication: true
```

Comment:
Can TRT-LLM handle multi-gpu inference easily? Then we should demonstrate that we can easily integrate that with 
```
parallelType: "custom"
parallelLevel: 4
```

Reply:
I tried standalone multi gpu inference, it didn't work for me. Although the model loaded on the 4 GPUs, the inference was hanging